src/datasources/db/v2/database-migrator.service.ts

```diff
@@ -0,0 +1,157 @@
+import { PostgresDatabaseService } from '@/datasources/db/v2/postgres-database.service';
+import {
+  LoggingService,
+  type ILoggingService,
+} from '@/logging/logging.interface';
+import { Inject, Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
```

Comment:
As before regarding usage of `@/config/configuration.module`.

Reply:
Unfortunately, Since we need `env` variables, it's not possible to use `configuration.module`.